lib/puppet/type/keystore.rb

```diff
@@ -0,0 +1,55 @@
+Puppet::Type.newtype(:keystore) do
+  desc 'generates an empty pkcs12 keystore'
```

Comment:
Should this be?

```suggestion
  @doc = 'generates an empty pkcs12 keystore'
```

I see it done this way in the puppet type development documentation (while the parameters still use `desc`). What is the difference?

Reply:
Attempting to quickly look through the code, I think this is just a matter of using a method vs. setting an instance variable:

https://github.com/puppetlabs/puppet/blob/67ad21b64237eb4f808844bdaa3c2b0e4cdef95c/lib/puppet/util/docs.rb#L4-L6

Using the `desc` method seems more robust from an API stand-point. And all our other types do this as well so there is consistency in use.

Reply:
OK, thanks for clearing that up. The method is there to be used, so let's use it then.